profile/main.go

```diff
@@ -389,20 +416,20 @@ func handler(request events.APIGatewayProxyRequest) (events.APIGatewayProxyRespo
 		if err != nil {
 			log.Fatalf("Failed to iterate: %v", err)
 		}
+		totalProfilesChecked += 1
 		var userId string = doc.Ref.ID
 		var userUrl string
 		var chaincode string
-		
 		if str, ok := doc.Data()["profileURL"].(string); ok {
 			userUrl = str
 		} else {
+			profilesSkipped.ProfileURL = append(profilesSkipped.ProfileURL, userId)
+			logProfileSkipped(client, ctx, userId, "Profile URL not available")
```

Comment:
We can make a global map[string]string for all the constants like status names, skipping reasons, and others.

var Constants map[string]string = map[string]string{
  "constantName": "constantValue",
  ...
}